pipelines/managed/e2e/README.md

```diff
@@ -18,13 +18,16 @@ affected by RHTAP services or which results could affect the RHTAP workflow.
 | postCleanUp                     | Cleans up workspace after finishing executing the pipeline                                                                         | Yes      | true                                                      |
 | verify_ec_task_bundle           | The location of the bundle containing the verify-enterprise-contract task                                                          | No       | -                                                         |
 | verify_ec_task_git_revision     | The git revision to be used when consuming the verify-conforma task                                                                | No       | -                                                         |
-| taskGitUrl                      | The url to the git repo where the release-service-catalog tasks to be used are stored                                              | Yes      | https://github.com/konflux-ci/release-service-catalog.git |
+| taskGitUrl                      | The url to the git repo where the release-service-catalog tasks to be used are stored                                              | No        | "" |
 | taskGitRevision                 | The revision in the taskGitUrl repo to be used                                                                                     | No       | -                                                         |
 
 ## Changes in 3.2.0
 * Add new parameter `verify_ec_task_git_revision` needed for consuming the verify-conforma task
   via git resolver
 
+## Changes in 3.2.0
+* Update all tasks that now support trusted artifacts to specify the stepAction* parameters
```

Comment:
This is also a nit pick, fine to leave as is IMO, but you could like `s/specify the stepAction* parameters/specify the taskGit* parameters for the step action resolvers/g` the READMEs